dramatiq/abortable/backend.py

```diff
@@ -0,0 +1,12 @@
+class EventBackend:
+    """ABC for event backends.
```

Comment:
Might as well use the [abc lib](https://docs.python.org/3/library/abc.html)?

Reply:
Rest of the projet don't, I prefer to stay consistent.

Reply:
@isra17 ,

Finally the most awaited feature seems to be available soon.
**Can you please give an example for your code how to use the abort functionality to terminate a long running task (message)?**